pulser-core/pulser/backend/abc.py

```diff
@@ -53,6 +53,12 @@ def validate_sequence(
         if not mimic_qpu:
             return
 
+        if sequence.is_empty:
+            raise ValueError(
+                "'sequence' should not be empty, please add content to it's "
+                "schedule."
```

Comment:
The notion of "schedule" is internal, how about
```suggestion
                "'sequence' should not be empty, please add an instruction "
                "to a declared channel."
```